docs/book/src/component-config-tutorial/testdata/project/Makefile

```diff
@@ -11,6 +11,10 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# CONTAINER_TOOL defines the container tool to be used for building images.
+# You can use docker or podman
+CONTAINER_TOOL ?= docker
```

Comment:
I've seen this pattern be successful and useful in other projects. It uses podman if it's installed, otherwise defaults to docker. That seems to match people's usage fairly well, because typically the only reason someone appears to have both installed is if they use podman and have some sort of alias of docker to podman (see `dnf info podman-docker` for example).

```
CONTAINER_TOOL=$( command -v podman &> /dev/null && echo "podman" || echo "docker")
```